pkgs/build-support/rust/default.nix

```diff
@@ -86,6 +86,10 @@ in stdenv.mkDerivation (args // {
     ${stdenv.lib.optionalString (stdenv.buildPlatform.config != stdenv.hostPlatform.config) ''
     [target."${stdenv.hostPlatform.config}"]
     "linker" = "${ccForHost}"
+    ${stdenv.lib.optionalString (stdenv.hostPlatform.config == "aarch64-unknown-linux-musl") ''
+    # https://github.com/rust-lang/rust/issues/46651#issuecomment-433611633
```

Comment:
Can this comment be outside the quote so we can change it without causing rebuilds?